### DISCUSSION TITLE
Add branch for personal config

### DISCUSSION THREAD

**Discussion by Zabooby** [dyphire/mpv-config#79] [Ideas]
Would you be open to creating a branch for your personal config, to showcase what you actually use and excluding all the supplementary stuff added for users of the general config?

You don't have to update it regularly, maybe once a month. Also no need for any detailed descriptions or readme's.

**Comment by dyphire** [comment]
Ah, it seems that you have misunderstood the previous response. In fact, this repository is the configuration I am using, and I do not have the energy to maintain two different configurations, which is why the English branch has not been maintained for a long time.

Files that I no longer use but feel still meet certain requirements will be placed in the archive folder, while shaders that I do not use but for testing purposes will be placed in `shaders/other` folders. These types of files will not be actively maintained either.

If the above two folders are excluded, that is the configuration I am currently using.

Edit: There are indeed many shaders in the shader folder that I rarely use, which are kept for functional reasons.

**Reply by Zabooby** [reply]
Oh, sorry of the confusion, thanks for all the help.
